pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/KitUtil.java

```diff
@@ -224,6 +235,17 @@ public static void createLabel(List<KitRequestCreateLabel> kitsLabelTriggered, E
         DBUtil.updateBookmark(0, BOOKMARK_LABEL_CREATION_RUNNING);
     }
 
+    public static boolean kitRequestSettingsNotFound(KitRequestSettings kitRequestSettings) {
+        return (kitRequestSettings == null || (kitRequestSettings.getCarrierTo() == null && kitRequestSettings.getServiceTo() == null
+                && kitRequestSettings.getCarrierToId() == null && kitRequestSettings.getCarrierToAccountNumber() == null && kitRequestSettings.getCarrierReturnId() == null
+                && kitRequestSettings.getCarrierReturn() == null && kitRequestSettings.getServiceReturn() == null && kitRequestSettings.getCarrierRetrunAccountNumber() == null
+                && kitRequestSettings.getWeight() == null && kitRequestSettings.getLength() == null && kitRequestSettings.getHeight() == null && kitRequestSettings.getWeight() == null
+                && kitRequestSettings.getCollaboratorSampleTypeOverwrite() == null && kitRequestSettings.getCollaboratorParticipantLengthOverwrite() == null && kitRequestSettings.getReturnName() == null
+                && kitRequestSettings.getReturnStreet1() == null && kitRequestSettings.getReturnStreet2() == null && kitRequestSettings.getReturnCity() == null && kitRequestSettings.getReturnZip() == null
+                && kitRequestSettings.getReturnState() == null && kitRequestSettings.getReturnCountry() == null && kitRequestSettings.getPhone() == null && kitRequestSettings.getDisplayName() == null
+                && kitRequestSettings.getExternalShipper() == null && kitRequestSettings.getExternalShipperKitName() == null));
```

Comment:
Not all values will be null, so I found the ones that would mean that the query went wrong. if one of these values has a value it could mean that it's not a sub kits problem because the joins between tables are working. Also we should not look at empty strings because they would also mean the value was found and only the config was set in a way that a value is null (hence the comparison to null instead of StringUtils.isBlank())